scripts/create_tables.sql

```diff
@@ -1 +1,38 @@
 -- create doctors, patients, and appointments tables
+--doctors
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING(128),
+  given_name CHARACTER VARYING(128),
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  specialty CHARACTER VARYING(128),
+
+  doctor_id INTEGER REFERENCES doctors(id),
+  patient_id INTEGER REFERENCES patients(id)
```

Comment:
Similarly, why is there a column for `patient_id` in the `doctors` table. Does each doctor belong to only one patient?